.github/ISSUE_TEMPLATE/support-request.yml

```
@@ -0,0 +1,30 @@
+name: 🙋 Ask a question
+description: >
+  Ask a question about or get help with Swift OpenAPI Generator. Beginner
+  questions welcome!
+labels: [kind/support, status/triage]
+body:
+  - type: markdown
+    attributes:
+      value: >
+        This repository hosts issues for the Swift OpenAPI generator, the Swift
+        OpenAPI runtime library, the Swift OpenAPI URLSession transport library,
+        and the AsyncHTTPClient transport library.
+        It does *not* track feedback on Xcode and other closed source Apple
+        developer software such as URLSession itself; please direct it to
+        [Feedback Assistant](https://developer.apple.com/bug-reporting) instead.
+
+        ___
+
+        Please use this issue template to ask a question.
+
+        You may also like to check out our
+        [FAQ](https://swiftpackageindex.com/apple/swift-openapi-generator/documentation/swift-openapi-generator/useful-openapi-patterns)
+        for solutions to commonly-asked questions and common workflows.
```

```suggestion
        for solutions to commonly asked questions and common workflows.
```

I believe a hyphen is not needed after -ly?

---

Fixed.